index.html

```diff
@@ -1,14 +1,33 @@
-Jovany Cortes Marure
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>JovaCov</title>
+        <meta charset="UTF-8">
+        <meta name="Description" content="Jovanys Personal Site">
+        <meta name="author" content="Jovany Cortes Marure">
+        <h1>Jovany Cortes Marure</h1>
```

Comment:
Note that `<meta>` tags are not displayed on the page but provide information to the browser or search engine about your page. Therefore, <h1>Your name</h1> should be moved inside the `<body>` tags.